File: python/darts/darts.py
import math

def score(x, y):
    distance_to_center = math.sqrt(x**2 + y**2)
    
    if distance_to_center <= 1:
        return 10
    elif distance_to_center <= 5:
        return 5
    elif distance_to_center <= 10:
        return 1
    else:
        return 0

File: java/darts/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double hypotenuse = Math.sqrt(Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2));
        return hypotenuse > 10 ? 0 : hypotenuse > 5 ? 1 : hypotenuse > 1 ? 5 : 10;
    }
}
